source/connect.txt

```diff
@@ -49,68 +49,69 @@ Connect
 
 When you open |compass-short|, an initial connection dialog appears:
 
-.. figure:: /images/compass/paste-connection-string.png
+.. figure:: /images/compass/new-connection.png
    :figwidth: 690px
-   :alt: Connection dialog for pasting your connection string
+   :alt: New connection dialog for pasting your connection string
 
-|compass-short| provides two methods to connect to your deployment,
-either providing your deployment connection string or filling in your
-deployment information in specific fields.
+|compass-short| provides multiple methods to connect to your deployment,
+either providing your deployment connection string or through Advanced 
+Connection Options. 
```

Comment:
After this paragraph, consider adding a quick sentence explaining what the advanced options consist of. For example:

"Advanced connection options allow you to specify authentication, TLS/SSL, and SSH connection options. To learn more, see (link to the Advanced Connection Options section further down on this page)."

Reply:
Done.